IND/MarketIndices/BasketIndices.rdf

```diff
@@ -299,7 +299,7 @@
 		<rdfs:label xml:lang="en">has duration of protection</rdfs:label>
 		<rdfs:domain rdf:resource="&fibo-ind-mkt-bas;CreditIndex"/>
 		<rdfs:range rdf:resource="&fibo-fnd-dt-fd;ExplicitDuration"/>
-		<skos:definition xml:lang="en">indicates the lifespan of protection, from the release date to the scheduled maturity date for the index</skos:definition>
+		<skos:definition xml:lang="en">indicates the lifespan of protection against default, from the release date to the scheduled maturity date for the index</skos:definition>
```

Comment:
I'm even more unclear:
- A CreditIndex is a basket of credit instruments each of which might have a maturity date, but I don't see that the index as a whole would have a maturity date. It could only have an average duration e.g. an index comprising a set of 10 year notes (generally each bought at separate times hence with a different maturity date) that are replaced when they mature
- I don't see how the index as a whole could default.
- this property seems redundant if it's always from the release date to the maturity date.

Maybe the domain should be CreditDefaultSwapIndex rather than CreditIndex? See https://www.investopedia.com/terms/d/dowjonescdx.asp#:~:text=The%20CDX%20index%20rolls%20over,leave%20the%20index%20as%20appropriate. where the duration seems to be 6 months.

Reply:
Can we address this in a separate issue?  It is really a very small part of this one, and likely requires some research.

Reply:
I'll just eliminate it.  The property is not used anywhere.  I think it was part of the original ontology MB created.  If we need it down the road we can add it back.

Reply:
Deprecated the property as mentioned above